tests/test_fxa_test_account.py

```diff
@@ -5,26 +5,42 @@
 import re
 
 import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
 
-from fxapom.fxapom import AccountNotFoundException, DEV_URL, FxATestAccount
+from fxapom.fxapom import DEV_URL, FxATestAccount, WebDriverFxA
 
 
-@pytest.mark.skip_selenium
 class TestFxATestAccount(object):
 
+    _fxa_unknown_account_error_locator = (By.CSS_SELECTOR, '#main-content div.error')
+
+    @pytest.mark.skip_selenium
     def test_default_environment_should_be_dev(self):
         account = FxATestAccount()
         assert DEV_URL == account.url
 
+    @pytest.mark.skip_selenium
     def test_create_new_account(self, account):
         assert account.is_verified
         # Test logging in - will throw an exception if log in fails
         account.login()
 
+    @pytest.mark.skip_selenium
     def test_new_account_pw_does_not_contain_numbers(self, dev_account):
         assert re.search(r'\d', dev_account.password) is None
```

Comment:
It's because it's only really needed on one evn, as there is only one path for the code that is executed to set the password regardless of the env. I suppose there is no reason that we shouldn't also run it against prod, other than the fact that it's unnecessary, but that is based on the current implementation. Although I cannot imagine that part of the code changing, I suppose it's possible at some point that different envs might encounter different code paths, so to future-proof against coverage loss we should probably run in on both. I'll update that.